mobile_config/src/gateway_service.rs

```diff
@@ -163,14 +163,18 @@ impl mobile_config::Gateway for GatewayService {
         let (tx, rx) = tokio::sync::mpsc::channel(100);
 
         let device_types: Vec<DeviceType> = request.device_types().map(|v| v.into()).collect();
+        let min_refreshed_at = DateTime::<Utc>::from_timestamp(request.min_refreshed_at as i64, 0)
```

Comment:
are we relying on this always defaulting to 0 in the proto when omitted, which should always decode to 1970-1-1? we could also `.unwrap_or_else(|| DateTime::UNIX_EPOCH)` which would have the equivalent result but i can see the argument for not silently defaulting to 1970-1-1 if the caller explicitly supplies an i64 timestamp that happens to be invalid

Reply:
> are we relying on this always defaulting to 0 in the proto when omitted, which should always decode to 1970-1-1? 

yes we are. I've left comment in [proto file](https://github.com/helium/proto/pull/429/files#diff-0351a6685cffa248fdc4ecd2d7188bf6fff91f11f905ce55fbe1107cec434a15R84) about this. 

> we could also .unwrap_or_else(|| DateTime::UNIX_EPOCH) which would have the equivalent result but i can see the argument for not silently defaulting to 1970-1-1 if the caller explicitly supplies an i64 timestamp that happens to be invalid

I don't see how we can use it in this case. Since `min_refreshed_at` is not an optional field in terms of proto file. 
If 0 was passed this is not an error. So, as for me `unwrap_or_else` doesn't make sense in this case.